### PR TITLE
OEL-2069: Gallery paragraph.

### PR DESCRIPTION
**Pull request by brummbar**

### CODE REVIEW

File: modules/oe_paragraphs_gallery/config/install/field.field.paragraph.oe_gallery.field_oe_plain_text_long.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    - field.storage.paragraph.field_oe_plain_text_long
+    - paragraphs.paragraphs_type.oe_gallery
+id: paragraph.oe_gallery.field_oe_plain_text_long
+field_name: field_oe_plain_text_long
+entity_type: paragraph
+bundle: oe_gallery
+label: Description
+description: ''
+required: false
+translatable: true
```
Comment (drishu) [inline modules/oe_paragraphs_gallery/config/install/field.field.paragraph.oe_gallery.field_oe_plain_text_long.yml:14]: In other components field instances we don't make translatable, but in paragraphs we do, was just wondering about it

File: modules/oe_paragraphs_gallery/config/install/field.field.paragraph.oe_gallery.field_oe_medias.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+dependencies:
+  config:
+    - field.storage.paragraph.field_oe_medias
+    - media.type.av_portal_photo
+    - media.type.av_portal_video
+    - media.type.image
+    - media.type.remote_video
+    - paragraphs.paragraphs_type.oe_gallery
+id: paragraph.oe_gallery.field_oe_medias
+field_name: field_oe_medias
```
Comment (drishu) [inline modules/oe_paragraphs_gallery/config/install/field.field.paragraph.oe_gallery.field_oe_medias.yml:12]: we agreed to field_oe_gallery_medias

**Review by drishu — CHANGES_REQUESTED** [review]

File: modules/oe_paragraphs_gallery/config/install/field.field.paragraph.oe_gallery.field_oe_plain_text_long.yml (hunk 1 above)
Comment (brummbar) [reply modules/oe_paragraphs_gallery/config/install/field.field.paragraph.oe_gallery.field_oe_plain_text_long.yml:14]: Uhm it depends on what it is the field and the scope.

Comment (brummbar) [reply modules/oe_paragraphs_gallery/config/install/field.field.paragraph.oe_gallery.field_oe_plain_text_long.yml:14]: Usually text fields are translatable, but "value" fields like boolean toggles or entity references are not.

**Review by drishu — APPROVED** [review]

**Review by drishu — APPROVED** [review]

**Review by upchuk — APPROVED** [review]

**Pull request merged**